api/openapi-spec/swagger.json

```diff
@@ -19434,7 +19434,7 @@
   },
   "info": {
     "title": "Arktos",
-    "version": "v0.2"
+    "version": "v2.7.4"
```

Comment:
Please don't change this value. v0.2 is current Arktos release version

Reply:
this is changed by make update. I didn't change it manually.